application/forms/Command/Object/ScheduleCheckForm.php

```diff
@@ -109,26 +111,25 @@ protected function assembleSubmitButton()
         (new IcingaFormDecorator())->decorate($this->getElement('btn_submit'));
     }
 
-    /**
-     * @return ?ScheduleCheckCommand
-     */
-    protected function getCommand(Model $object)
+    protected function getCommands(Traversable $objects): Generator
     {
-        if (
-            ! $this->isGrantedOn('icingadb/command/schedule-check', $object)
-            && (
-                ! $object->active_checks_enabled
-                || ! $this->isGrantedOn('icingadb/command/schedule-check/active-only', $object)
-            )
-        ) {
-            return null;
-        }
+        list($firstGranted, $allGranted) = $this->copyFirstItem(new CallbackFilterIterator(
+            $objects instanceof Iterator ? $objects : new IteratorIterator($objects),
+            function (Model $object): bool {
+                return $this->isGrantedOn('icingadb/command/schedule-check', $object) || (
+                    $object->active_checks_enabled
+                    && $this->isGrantedOn('icingadb/command/schedule-check/active-only', $object)
+                );
+            }
+        ));
```

Comment:
... **and** a whiteboard for this. 😅

![8F58897B-41A5-4D25-8BE4-E8A6FFB17186](https://user-images.githubusercontent.com/6053266/222185988-01b20bf0-e4b1-48b5-a558-cd75cc583ff3.jpeg)